botcity/base/bot.py

```diff
@@ -52,6 +52,9 @@ def _search_image_file(self, label):
                 return str(f.absolute())
         return None
 
+    def _image_path_as_image(self, path):
+        return Image.open(path)
```

Comment:
Here we are missing a validation of the variable `path`. Since it can be `None` we need to check and properly return the value `None` if that is the case.

Reply:
Addressed via https://github.com/botcity-dev/botcity-framework-base-python/pull/1/commits/c396d4d4cd033261810fa11927fe8a0a7d794010